src/controllers/TalksController.php

```diff
@@ -320,10 +306,18 @@ public function removeTrackFromTalk(Request $request, PDO $db)
      *
      * @return TalkModelCollection
      */
-    protected function getTalkById($db, $request, $talk_id)
-    {
-        $mapper = new TalkMapper($db, $request);
-        $talk   = $mapper->getTalkById($talk_id);
+    protected function getTalkById(
```

Comment:
Rewriting this method is not necessary for the scope of the PR. so please add it as a separate PR.